FlowCal/mef.py

```diff
@@ -441,9 +510,9 @@ def get_transform_fxn(data_beads,
         Results of the subpopulation selection step, containing the
         following fields:
 
-        channel : list
+        au : list
             The fluorescence values of each selected subpopulation in
-            channel units, for each channel in `mef_channels`.
+            arbitrary units, for each channel in `mef_channels`.
```

Comment:
again, maybe rebrand as `rfi`? to be more precise / clear that we should only be dealing with RFI values at this point